### ISSUE TITLE
Error in local CI

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-server#1462]
```console
$ poetry run pytest -x -v -s --log-cli-level info tests/v2/08_full_workflow/test_full_workflow_slurm_v2.py::test_full_workflow_slurm
===================================================================================================================== test session starts =====================================================================================================================
platform linux -- Python 3.10.12, pytest-7.4.4, pluggy-1.5.0 -- ..../fractal-server-cyevISt_-py3.10/bin/python
cachedir: .pytest_cache
rootdir: ....
configfile: pyproject.toml
plugins: docker-2.2.0, subprocess-1.5.0, devtools-0.12.2, anyio-4.3.0, asyncio-0.21.2
asyncio: mode=auto
collected 1 item                                                                                                                                                                                                                                              

tests/v2/08_full_workflow/test_full_workflow_slurm_v2.py::test_full_workflow_slurm 2024-05-03 16:34:54,186 - fractal_server.app.db - WARNING - SQLite is supported (for version >=3.37) but discouraged in production. Given its partial support for ForeignKey constraints, database consistency cannot be guaranteed.
2024-05-03 16:34:54,190 - fractal_server.app.db - WARNING - SQLite is supported (for version >=3.37) but discouraged in production. Given its partial support for ForeignKey constraints, database consistency cannot be guaranteed.

----------------------------------------------------------------------------------------------------------------------- live log setup ------------------------------------------------------------------------------------------------------------------------
WARNING  root:fixtures_slurm.py:127 docker_compose_project_name='pytest105249'
WARNING  root:fixtures_slurm.py:128 slurm_container='pytest105249-slurm-docker-master-1'
WARNING  root:fixtures_tasks_v2.py:66 Actual task collection
2024-05-03 16:34:54,291 - collect_tasks_pip - DEBUG - Task-collection endpoint: start background collection and return state
2024-05-03 16:34:54,291 - fractal_tasks_mock - DEBUG - Start background task collection
2024-05-03 16:34:54,291 - fractal_tasks_mock - DEBUG - package: fractal_tasks_mock
2024-05-03 16:34:54,291 - fractal_tasks_mock - DEBUG - package_version: 0.0.1
2024-05-03 16:34:54,291 - fractal_tasks_mock - DEBUG - package_extras: None
2024-05-03 16:34:54,291 - fractal_tasks_mock - DEBUG - python_version: 3.9
2024-05-03 16:34:54,291 - fractal_tasks_mock - DEBUG - pinned_package_versions: None
2024-05-03 16:34:54,291 - fractal_tasks_mock - DEBUG - package_name: fractal-tasks-mock
2024-05-03 16:34:54,292 - fractal_tasks_mock - DEBUG - package_path: ..../fractal-server/tests/v2/fractal_tasks_mock/dist/fractal_tasks_mock-0.0.1-py3-none-any.whl
2024-05-03 16:34:54,293 - fractal_tasks_mock - DEBUG - Task-collection status: installing
2024-05-03 16:34:54,294 - fractal_tasks_mock - DEBUG - Creating venv and installing package
2024-05-03 16:34:54,294 - fractal_tasks_mock - DEBUG - [_init_venv] path=PosixPath('..../pytest-6/FRACTAL_TASKS_DIR/.fractal/fractal-tasks-mock0.0.1')
2024-05-03 16:34:54,294 - fractal_tasks_mock - DEBUG - [_init_venv] interpreter='/usr/bin/python3.9'
2024-05-03 16:34:54,294 - fractal_tasks_mock - ERROR - Task manifest loading failed
2024-05-03 16:34:54,295 - fractal_tasks_mock - DEBUG - Task-collection status: fail
2024-05-03 16:34:54,295 - fractal_tasks_mock - INFO - Background collection failed. Original error: 
2024-05-03 16:34:54,297 - fractal_tasks_mock - INFO - Now deleting temporary folder ..../pytest-6/FRACTAL_TASKS_DIR/.fractal/fractal-tasks-mock0.0.1
2024-05-03 16:34:54,297 - fractal_tasks_mock - INFO - Temporary folder deleted
2024-05-03 16:34:54,299 - check_collection_status - DEBUG - Querying state for state.id=1
ERROR

...
```

**Comment by tcompa** [comment]
We fond out with @ychiucco that this can be solved by setting the proper order of execution for fixtures.
* The relink-python-interpreter-v2 needs fractal-tasks-mock-venv
* fractal-tasks-mock-venv must be executed before monkey-slurm (we are not 100% sure here, but the most likely reason is that otherwise the `subprocess.run` statements of the task collection are run from within the container - and they fail for unclear reasons)

**Comment by tcompa** [comment]
Fixed with #1476
